packages/runtime/container-runtime/src/garbageCollectionTombstoneUtils.ts

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ITelemetryGenericEvent } from "@fluidframework/common-definitions";
+import { packagePathToTelemetryProperty } from "@fluidframework/runtime-utils";
+import { MonitoringContext } from "@fluidframework/telemetry-utils";
+import { throwOnTombstoneUsageKey } from "./garbageCollectionConstants";
+
+/**
+ * Decides whether or not to send an error event or a generic event for gc tombstone scenarios
+ *
+ * Adds isSummarizerClient, packagePath, and error to telemetry properties.
+ */
+export function sendGCTombstoneEvent(
+    mc: MonitoringContext,
+    event: ITelemetryGenericEvent,
+    isSummarizerClient: boolean,
+    packagePath: readonly string[] | undefined,
+    error?: any,
+) {
+    const throwOnTombstoneUsage = event.throwOnTombstoneUsage = mc.config.getBoolean(throwOnTombstoneUsageKey) ?? false;
+    event.pkg = packagePathToTelemetryProperty(packagePath);
+    event.isSummarizerClient = isSummarizerClient;
+    if(throwOnTombstoneUsage) {
+        mc.logger.sendErrorEvent(event, error);
+    } else {
+        mc.logger.sendTelemetryEvent(
+            {
+                ...event,
+                ...error,
```

Comment:
@tyler-cai-microsoft - Instead of this you can just pass `error` to `sendTelemetryEvent` same as `sendErrorEvent`.  This doesn't do what you want anyway - message and stack are not included in `...error` (shocking I know)